### PR TITLE
eos: Update deleted operation for eos_acls and eos_static_routes

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY
This PR refactors the deleted state operation for eos_acls and eos_static_routes

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
eos_acls, eos_static_routes

### CODE REVIEW

File: plugins/module_utils/network/eos/config/acls/acls.py [hunk 1]
```diff
@@ -341,8 +337,12 @@ def _state_deleted(want, have):
                     commands.append(command)
         else:
             for w in want:
-                return_command = del_commands(w, have)
-                commands.append(return_command)
+                return_command = del_commands(w, have, True)
+                if "Warn" in return_command:
+                    self._module.warn(" ".join(return_command.split()[1:]))
```
Comment (Qalthos) [inline plugins/module_utils/network/eos/config/acls/acls.py:342]:
```suggestion
                if return_command.startswith("Warn "):
                    self._module.warn(return_command.split(" ", 1)[1])
```

Alternately,

```suggestion
                if return_command[:5] == "Warn ":
                    self._module.warn(return_command[5:])
```

**Comment by GomathiselviS** [comment]
recheck

File: tests/integration/targets/eos_acls/tasks/main.yaml [hunk 2]
```diff
@@ -2,7 +2,3 @@
 - include: cli.yaml
   tags:
     - cli
```
Comment (pabelanger) [inline tests/integration/targets/eos_acls/tasks/main.yaml:4]: this needs to be network_cli now, we've updated our jobs.

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Pull request merged**